e2e/query-editor.spec.ts

```diff
@@ -378,6 +376,7 @@ test.describe('Prometheus query editor', () => {
       await explorePage.runQuery();
     });
 */
+    // Metrics explorer option was removed https://github.com/grafana/grafana/pull/93262
```

Comment:
maybe we can replace this with `//TODO query for metrics explorer button once prometheusUsesCombobox is GA and enabled by default` just so we don't forget about it